users/models.py

```diff
@@ -260,10 +260,24 @@ def get_content_object_by_model(self, model_or_instance):
 
         return session_element.content_object
 
-    def end(self):
+    def end(self, send_logout_to_apis=False, request=None):
```

Comment:
I'd like to keep the default behavior not to call external services. As it's not really common in Django models.

Reply:
> ...call external services. As it's not really common in Django models

This text makes me ponder that is calling external services from a Django model the correct thing to do in **any** situation? Should this "ending a `TunnistamoSession`" functionality be extracted out of the model somehow?

Reply:
That's one thing I meant when I said I'd like to discuss this feature. But yeah, maybe. But it would get always called after the Tunnistamo Session is ended. Now the session is ended in two places, so it would get called in two different places. Of course if there would be more places in the future, one needs to remember to call the "API log out functionality" in those too.

Reply:
I propose there should be a "session ending" function somewhere, that does more or less all what is now in `TunnistamoSession.end`. I'm just having a hard time trying to come up with the correct location for it...

Reply:
Concluded that we'll let this be as it is, as no "best way" came about.